test/unit/test_estimator.py

```diff
@@ -38,15 +37,12 @@ def test_unsupported_values_for_estimator_options(self):
             {"resilience_level": 4, "optimization_level": 3},
             {"optimization_level": 4, "resilience_level": 2},
         ]
-        with Session(
-            service=FakeRuntimeService(channel="ibm_quantum", token="abc"),
-            backend="common_backend",
-        ) as session:
```

Comment:
I will add these back - removed them originally because I added a check to see if the backend was a simulator. The IQP API will be updated so this check is no longer necessary